CODEOWNERS

```diff
@@ -5,5 +5,22 @@
 # Order is important: later rules override preceding rules
 
 # These owners will be the default owners for everything in the repo unless a later match takes precedence
-* @mesosphere-eng-noctilucent
-/stable/dispatch @devx
+* @mesosphere/sig-ksphere-cluster @mesosphere/sig-ksphere-multi-cluster
```

Comment:
hm, what about not having a default at all?

Reply:
Well, there's still files not covered here so we actually need a fallback I think.

Reply:
Perhaps we need a triage group to assign it appropriately.